health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/CampaignSummary.js

```
@@ -107,25 +107,23 @@ function reverseDeliveryRemap(data, t) {
 
   const parseConditionAndCreateRules = (condition, ruleKey, products) => {
     const conditionParts = condition.split("and").map((part) => part.trim());
-    let rules = [];
-
+    let attributes = [];
```

Biome

[error] This let declares a variable that is only assigned once.

'attributes' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)